core/core_integrations/messenger.js

```diff
@@ -0,0 +1,198 @@
+var fs = require('fs'),
+    path = require('path'),
+    express = require('express'),
+    bodyParser = require('body-parser'),
+    request = require('request'),
+    https = require('https'),
+    shim = require('../shim.js'),
+    http = null,
+    api = null,
+
+	getNameForId = function(id, callback) {
+		if (!exports.config.users) {
+			exports.config.users = {};
+		}
+
+		if (exports.config.users[id]) {
+			return callback(exports.config.users[id].name);
+		}
+
+		request({
+			url: 'https://graph.facebook.com/v2.6/' + id,
+			qs: {access_token: exports.config.token, fields: 'first_name,last_name'},
+			method: 'GET'
+		},
+		function(error, response, body) {
+			if (error) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2575812713/issues/source?bid=3282987&fileBranchId=3288129#l26)